tests/authorizations/test_staff_tenant_authorization.py

```diff
@@ -0,0 +1,57 @@
+import pytest
+
+
+def endpoint():
+    return "/api/staff-tenants"
```

Comment:
I think to keep things clean and consistent, let's move this into a `setup` method like in the other files:
```python
class TestStaffTenantAuthorizations:
    def setup(self):
        self.endpoint = "/api/staff-tenants"
    ....
````

Reply:
Just moved stuff around. Take a look!